interface/forms/dictation/report.php

```diff
@@ -1,7 +1,29 @@
 <?php
-//------------Forms generated from formsWiz
+/** 
+ *  Dictation report for display 
+ * 
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ * @copyright Copyright (c) 2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * 
+ */
+
 include_once(dirname(__FILE__).'/../../globals.php');
 include_once($GLOBALS["srcdir"]."/api.inc");
+require_once("../../../public/assets/htmlpurifier-4.9.2/library/HTMLPurifier.auto.php");
```

Comment:
After you rebase code to most recent codebase, then you can remove the above line.